### PR TITLE
MFA Documentation

### PR DESCRIPTION
**Pull request by lfbrock**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -1055,6 +1055,41 @@ Login Button Text
...
+
+Enable Multi-factor Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**True**: When true, users with LDAP and email authentication will be given the option to require a phone-based passcode, in addition to their password-based authentication, to sign-in to the Mattermost server. Specifically, they will be asked to download the `Google Authenticator <https://en.wikipedia.org/wiki/Google_Authenticator>`_ app to their iOS or Android mobile device, connect the app with their account, and then enter a passcode generated by the app on their phone whenever they log in to the Mattermost server.
+
+**False**: Multi-factor authentication is optional.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableMultifactorAuthentication": true`` with options ``true`` and ``false`` for above settings respectively.           |
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: I think default is 'false'

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -1055,6 +1055,41 @@ Login Button Text
...
+
+Enforce Multi-factor Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**True**: When true, users with LDAP and email login will be required to set up multi-factor authentication.
+
+**False**: Multi-factor authentication is disabled.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnforceMultifactorAuthentication": true`` with options ``true`` and ``false`` for above settings respectively.           |
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: Here also, the default is 'false'

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -1055,6 +1055,41 @@ Login Button Text
...
+
+If you choose to run Mattermost outside your private network, bypassing your existing security protocols, it is recommended you upgrade to Mattermost Enterprise Edition to set up a multi-factor authentication service specifically for accessing Mattermost.
+
+Enable Multi-factor Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**True**: When true, users with LDAP and email authentication will be given the option to require a phone-based passcode, in addition to their password-based authentication, to sign-in to the Mattermost server. Specifically, they will be asked to download the `Google Authenticator <https://en.wikipedia.org/wiki/Google_Authenticator>`_ app to their iOS or Android mobile device, connect the app with their account, and then enter a passcode generated by the app on their phone whenever they log in to the Mattermost server.
+
+**False**: Multi-factor authentication is optional.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: I think you got this mixed up with the setting below -- here MFA is disabled if the setting is set to false

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -1055,6 +1055,41 @@ Login Button Text
...
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableMultifactorAuthentication": true`` with options ``true`` and ``false`` for above settings respectively.           |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Enforce Multi-factor Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**True**: When true, users with LDAP and email login will be required to set up multi-factor authentication.
+
+**False**: Multi-factor authentication is disabled.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: ... and here MFA is optional, if `EnforceMultifactorAuthentication` is set to false

File: source/administration/config-settings.rst (hunk 1 above)
Comment (lfbrock) [inline source/administration/config-settings.rst]: Ah yeah that's incorrect, I think I just copied it from the old doc

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -1055,6 +1055,41 @@ Login Button Text
...
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableMultifactorAuthentication": false`` with options ``true`` and ``false`` for above settings respectively.           |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Enforce Multi-factor Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**True**: When true, users with LDAP and email login will be required to set up multi-factor authentication.
```
Comment (esethna) [inline source/administration/config-settings.rst]: We should probably state what happens when this is turned on. 

ie are all users without MFA logged out? what happens to sys admin account if MFA is not set up when they flip this switch? etc

Comment (lfbrock) [inline source/administration/config-settings.rst]: Updated, System Admin account is the same as all other users (redirected to set up page until set up is complete)

**Pull request merged**